### PR TITLE
Makefile changes to Pypi packaging

### PR DESCRIPTION
**Pull request by aravindavk**
`pip3 install twine` is required to upload to Pypi repository.

Along with the release,

```
$ export TWINE_PASSWORD=<secret>
$ KADALU_VERSION=0.4.0 make release
```

Only upload to pypi,

```
$ export TWINE_PASSWORD=<secret>
$ KADALU_VERSION=0.4.0 make pypi-upload
```

Signed-off-by: Aravinda VK <mail@aravindavk.in>

### CODE REVIEW

**Review by amarts — COMMENTED** [review]
LGTM..

Just curious. Considering this involves 'kadalu_quotad/' changes here, should we have that PR before?

**Comment by aravindavk** [comment]
> Just curious. Considering this involves 'kadalu_quotad/' changes here, should we have that PR before?

Sure. I will rebase once Quota PR merges.

File: Makefile [hunk 1]
```diff
@@ -78,10 +78,17 @@ prepare-release:
 		$(MAKE) build-containers
 endif
 
+pypi-build:
+	echo ${KADALU_VERSION} > cli/VERSION
+	cd cli; rm -rf dist; python3 setup.py sdist;
```
Comment (amarts) [inline Makefile:83]: I am thinking of one more line for kadalu_quotad here. Do you want to do that as a separate patch? or change in this itself?

File: Makefile [hunk 2]
```diff
@@ -78,10 +78,17 @@ prepare-release:
...
+
+pypi-upload: pypi-build
+	cd cli; twine upload --username kadalu dist/*
```
Comment (amarts) [inline Makefile:89]: same here, one more line for kadalu_quotad

File: cli/setup.py [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 
+def version():
+    with open("VERSION") as version_file:
+        return version_file.read().strip()
+
 
 setup(
     name="kubectl-kadalu",
-    version="0.1.0",
+    version=version(),
```
Comment (amarts) [inline cli/setup.py:10]: Looks like I needed to add this in quotad.py too.

**Comment by aravindavk** [comment]
Also included kadalu-quotad package.

**Review by amarts — APPROVED** [review]
